scene/gui/tab_container.cpp

```diff
@@ -417,7 +419,13 @@ void TabContainer::set_current_tab(int p_current) {
 	}
 
 	_change_notify("current_tab");
-	emit_signal("tab_changed", current);
+	if (pending_previous == current)
+		emit_signal("tab_selected", current);
```

Comment:
Yes, I didn't pay much attention to it, since I wanted minimal changes to previous commit. But functionality wise, it would be the same. I will change this now :)